blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -68,16 +69,20 @@ export class Activity extends Component {
             return null;
         }
 
+        const { capabilities } = this.props;
+        const isMultiBranchPipeline = capabilities[pipeline._class].contains(MULTIBRANCH_PIPELINE);
```

Comment:
Ohhh this makes me a little sad tbh .... referencing `_class` directly ... the `MULTIBRANCH_PIPELINE` constant is a bit ugly 😢 

What was there (`Pipeline.isMultibranch(pipeline)`) def seems better to me, but doesn't use the capabilities so I accept that that might not work as is, but surely it could be morphed into something like `Pipeline.isMultibranch(capabilities)` and centralize/hide logic like `capabilities[pipeline._class].contains(MULTIBRANCH_PIPELINE)`.